composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "umakantp/smarty-lint",
+    "description": "SmartyLint is a PHP tool that helps you find issues in your Smarty template files. It's designed to assist you in writing cleaner and more consistent Smarty templates",
+    "type": "project",
```

Comment:
Shouldn't this be a library?

Ref: https://getcomposer.org/doc/04-schema.md#type